demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -2395,18 +2358,17 @@ int RunOtaCoreHttpDemo( bool awsIotMqttMode,
                         const char * pIdentifier,
                         void * pNetworkServerInfo,
                         void * pNetworkCredentialInfo,
-                        const IotNetworkInterface_t * pNetworkInterface )
+                        const IotNetworkInterface_t * pxNetworkInterface )
 {
     ( void ) awsIotMqttMode;
     ( void ) pIdentifier;
     ( void ) pNetworkServerInfo;
     ( void ) pNetworkCredentialInfo;
-    ( void ) pNetworkInterface;
+    ( void ) pxNetworkInterface;
 
     /* Return error status. */
-    int returnStatus = EXIT_SUCCESS;
-
-    bool mqttInitialized = false;
+    BaseType_t xDemoStatus = pdPASS;
+    BaseType_t mqttInitialized = pdFALSE;
```

Comment:
Missing the x prefix

Reply:
Thanks updated in https://github.com/aws/amazon-freertos/pull/3148/commits/6bbe8a3c217e31fff82104a685ad811120104724